lib/ansible/modules/web_infrastructure/pmp.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Bernat Mut <bernat.mut@aireuropa.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+module: pmp
+version_added: "2.5"
+short_description: Create and delete resources in Password Manager Pro (PMP)
+description:
+  - This allows to create and delete a resource with an account inside Password Manager Pro (PMP).
+
```

Comment:
It would be helpful to add a [`notes`](https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#ansible-metadata-block) section that mentions to the user that they should use the lookup plugin for retrieving existing password records from Password Manager Pro.